### PR TITLE
[CLEANUP Beta] Remove EmberKeys (internally) continue to expose Ember…

### PR DESCRIPTION
**Pull request by stefanpenner**
….keys but as 1:1 with Object.keys [ES5-only]

(╯°□°）╯︵ sʎǝʞ¬

### CODE REVIEW

File: packages/ember-application/tests/system/logging_test.js [hunk 1]
```diff
@@ -7,11 +7,8 @@ import View from 'ember-views/views/view';
 import Controller from 'ember-runtime/controllers/controller';
 import Route from 'ember-routing/system/route';
 import RSVP from 'ember-runtime/ext/rsvp';
-import keys from 'ember-metal/keys';
 import compile from 'ember-template-compiler/system/compile';
 
-import 'ember-routing';
```
Comment (rwjblue) [inline packages/ember-application/tests/system/logging_test.js]: I believe that this is needed for side-effects.  Specifically, the run loop queues that are created come to mind (though I could be wrong).

File: packages/ember-metal/lib/main.js [hunk 2]
```diff
@@ -224,8 +223,8 @@ Ember._Cache = Cache;
 
 Ember.generateGuid    = generateGuid;
 Ember.GUID_KEY        = GUID_KEY;
-Ember.keys            = keys;
 Ember.create          = Object.create;
+Ember.keys            = Object.keys;
```
Comment (rwjblue) [inline packages/ember-metal/lib/main.js]: Can we use `Ember.deprecateFunc` here?

Something like:

```javascript
Ember.keys = Ember.deprecateFunc('Ember.keys has been deprecated, please use Object.keys instead.', Object.keys);
```

Comment (stefanpenner) [inline packages/ember-metal/lib/main.js]: future PR will deprecate these

File: packages/ember-application/tests/system/logging_test.js (hunk 1 above)
Comment (stefanpenner) [inline packages/ember-application/tests/system/logging_test.js]: good catch, although surprisingly no tests failed ;(

File: packages/ember-metal/lib/main.js (hunk 2 above)
Comment (rwjblue) [inline packages/ember-metal/lib/main.js]: kk, sounds god

**Pull request merged**